.github/workflows/php7.2.yml

```diff
@@ -0,0 +1,98 @@
+name: "Website build"
+
+on: ["push", "pull_request"]
+
+env:
+  PHP_VERSION: "7.2"
+
+jobs:
+  tests:
+
+    runs-on: "ubuntu-16.04"
+
+    services:
+      mysql:
+        image: "mysql:5.7"
+        env:
+          MYSQL_DATABASE: "doctrine_website_test"
+          MYSQL_ROOT_PASSWORD: "VdtLtifRh4gt37T"
```

Comment:
Should we use a Github secret here? We might as well use `unsecure` here :P

Reply:
True, but it is already hardcoded in travis file and in Git for some time only for the test environment. This will be fixed separately after #356 when I am able to use the Github Action secrets.